.github/workflows/publish.yml

```diff
@@ -41,14 +43,16 @@ jobs:
         id: meta
         with:
           images: protomaps/go-pmtiles
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
```

Comment:
There's an example of this here https://docs.docker.com/build/ci/github-actions/push-multi-registries/

Looks like you might also need this:

```yaml
      - name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v3
```

And you could push to multiple registries in one step with this:

```yaml
        with:
          context: .
          platforms: linux/amd64,linux/arm64
          push: true
          tags: |
            user/app:latest
            user/app:1.0.0
            ghcr.io/user/app:latest
            ghcr.io/user/app:1.0.0
```

Reply:
How do we determine the tags `latest` and `1.0.0` dynamically? https://github.com/protomaps/go-pmtiles/pull/139